orchagent/portsorch.cpp

```diff
@@ -444,6 +444,16 @@ void PortsOrch::removeDefaultBridgePorts()
         }
         if (attr.value.s32 == SAI_BRIDGE_PORT_TYPE_PORT)
         {
+            Port port;
+	    getPortByBridgePortId(bridge_port_list[i], port);
```

Comment:
The flushFdbEntries() member function has a parameter of Port so that we can display the portname in the SWSS logs. Displaying port-oids in SWSS logs may not be useful, that is why we are passing Port as parameter to flushFdbEntries. That is the reason here we have to convert to Port and back again in the function, it will be using m_bridge_port_id. This was one of our Internal PR comments. Will align the code in the next commit.

Reply:
for the log part, we can log from the caller.. but I feel it is better to avoid this port/bridge_port conversion..